src/components/HeaderDrawer/index.js

```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
 
 export default ({
   recentItems,
+  changeRecentItems,
   onClickHome,
   onCloseDrawer,
   drawerOpen,
   onOpenFile,
   onOpenRecentItem,
   onClickTemplate,
 }) => {
-  const c = useStyles()
-
+  const c = useStyles();
   const onDrop = useCallback((acceptedFiles) => {
     onOpenFile(acceptedFiles[0])
   }, [])
 
-  const { getRootProps, getInputProps } = useDropzone({ onDrop })
+  function onDeleteFile(i) {
+    console.log(i);
+    changeRecentItems( recentItems.filter(function(oneRecentFile){console.log(oneRecentFile.id);return oneRecentFile.id!==i}));
+    localStorage.removeItem('recentItems');
```

Comment:
you don't need to remove prior to the `setItem`

Reply:
Ok i will change this in this case. Sry for the beginner mistake im starting